torch/__init__.py

```diff
@@ -410,6 +411,22 @@ def is_deterministic():
     """
     return _C._get_deterministic()
 
+def set_warnOnlyOnce(b):
+    r"""Convert the TORCH_WARN_ONCE C++ macro into TORCH_WARN for tests, so
+    that warnings can be reliably tested for.
+
+    Args:
+        b (:class:`bool`): If False, force warnings to always be emitted
+                           If True, set to the default behaviour
+    """
+    _C._set_warnOnlyOnce(b)
+
+def is_warnOnlyOnce():
+    r"""Returns True if the global warnOnlyOnce flag is turned on. Refer to
+    :func:`torch._set_warnOnlyOnce` documentation for more details.
```

Comment:
I think there's an extra underscore here preventing the link from working. See here:

https://9986620-65600975-gh.circle-artifacts.com/0/docs/generated/torch.is_warnOnlyOnce.html

Reply:
fixed